src/App/Pages/Accounts/LoginApproveDeviceViewModel.cs

```diff
@@ -118,6 +125,15 @@ public async Task InitAsync()
             {
                 HandleException(ex);
             }
+
+            // TODO: Change this expression to, Appear if the browser is trusted and shared the key with the app
+            ContinueEnabled = !RequestAdminApprovalEnabled && !ApproveWithMasterPasswordEnabled && !ApproveWithMyOtherDeviceEnabled;
```

Comment:
❓  Is this TODO going to be addressed in this PR or in another one?

Reply:
This will be address in another PR.